### ISSUE TITLE
Search Zotero brings up nothing / can't see how to change Zotero ID

### ISSUE THREAD

**Issue by daniel-allington** [takluyver/cite2c#30]
I am running Jupyter 4.3.0 on Python 3.5.2, on Linux Minut 17.3 (64 bit), using Firefox and Opera as browsers. I have just installed cite2c using pip. The toolbar buttons have appeared in the notebook toolbar. When I first used one of them, the cite2c UI prompted me for my Zotero ID. I provided it. Now when I use the 'Insert citation' button, it brings up the 'Search Zotero' dialogue box. I type in some search terms. When I do this in Opera, it always tells me there are no matches - regardless of what the search terms were. When I do it in Firefox, I get no response at all. Either way, no citation is inserted.

It occurs to me that there may have been a problem in connecting to Zotero (e.g. I suppose it's possible that I entered the ID incorrectly, though there was no error message at the time), but I can't see any way of changing or resetting this and reconnecting.

**Comment by daniel-allington** [comment]
Ah okay, I see the main problem was the one raised in [issue #29]( https://github.com/takluyver/cite2c/issues/29). Privacy settings needed adjusting - a warning about that would have been good though.

However, I still can't see a way of changing the Zotero library that cite2c links to.

**Comment by takluyver** [comment]
Yep, this is still pretty prototype-y. Once it's got an ID, it doesn't currently have a way to change it other than looking in `~/.jupyter/nbconfig/`.

**Comment by daniel-allington** [comment]
Ah okay - but that's certainly better than nothing, so thanks for the tip!
